src/native/libs/build-native.proj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
 
   <PropertyGroup Condition="'$(TargetOS)' == 'wasi'">
-    <WASI_SDK_PATH Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/WASI-SDK-VERSION-25.0')">$([MSBuild]::NormalizeDirectory($(RepoRoot), 'src', 'mono', 'wasi', 'wasi-sdk'))</WASI_SDK_PATH>
+    <WASI_SDK_PATH Condition="'$(WASI_SDK_PATH)' == '' or !Exists('$(WASI_SDK_PATH)/VERSION25')">$([MSBuild]::NormalizeDirectory($(RepoRoot), 'src', 'mono', 'wasi', 'wasi-sdk'))</WASI_SDK_PATH>
```

Comment:
Why the change? It's best to include this marker file in one's local WASI SDK.

Reply:
I couldn't see how this used to work, but this sentinel name is the same as
https://github.com/dotnet/runtimelab/blob/9d025fa5e9d07e3e93061b4457ba6279b178df98/eng/pipelines/runtimelab/install-wasi-sdk.ps1#L36

Reply:
This looks to be the fallout from https://github.com/dotnet/runtime/pull/110654, so we just need to update `install-wasi-sdk.ps1` to use the new file name scheme.

Reply:
This is just sanity check, at some point our CI had too many versions on WASI-SDK and it was difficult to understand why it's wrong. 
The check implemented as simple file name is easier than reading the content of `VERSION` file. Because there are multiple places.

The better alternative is
https://github.com/dotnet/runtime/blob/37e4d45236e68946db9d264593aa31a9c00534bc/src/mono/wasi/build/WasiApp.targets#L103-L108

but we don't do that for in-tree builds.

Reply:
Thanks, updated `install-wasi-sdk.ps1` as suggested.